core/src/test/java/org/infinispan/stress/DataContainerStressTest.java

```diff
@@ -89,12 +89,7 @@ public boolean isEnabled() {
                     return false;
                  }
 
-                 @Override
-                 public void handleInMemoryExpiration(InternalCacheEntry entry, long currentTime) {
-
-                 }
-
-                 @Override
+               @Override
```

Comment:
Nitpick: spacing is off so slightly.

Reply:
Correction, that line was like that before, and only some of the lines in the inner class were aligned at 4 spaces. So I've changed all of them to 3 spaces.